### PR TITLE
AC Test Improvement - LRAC-10627 | master

### PR DESCRIPTION
**Pull request by ViniLopes20**
https://issues.liferay.com/browse/LRAC-10627
[Testray](https://testray.liferay.com/home/-/testray/case_results/1618894256?redirect=https%3A%2F%2Ftestray.liferay.com%2Fhome%2F-%2Ftestray%2Fcase_results%2Findex%3FtestrayCaseName%3DAssertLiferayDataSourceStatus%26testrayBuildId%3D1618893566%26testrayRunId%3D1618894140%26cur%3D1%26delta%3D200%26orderByCol%3Dstatus_sortable%26orderByType%3Dasc)

### CODE REVIEW

**Comment by ViniLopes20** [comment]
ci:test:sf

**Comment by ViniLopes20** [comment]
ci:stop

**Comment by ViniLopes20** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/AnalyticsCloudConnectionTest.testcase [hunk 1]
```diff
@@ -174,10 +174,70 @@ definition {
 	test AssertLiferayDataSourceStatus {
 		property portal.release = "true";
 
-		task ("Connect DXP to AC") {
-			var propertyName = ACDXPSettings.connectDXPtoAnalyticsCloud();
+		var list1 = ListUtil.newListFromString("Current Status,Synced Sites,Synced Contacts");
+		var statusSyncSite = ListUtil.newListFromString("Active,Configured,Unconfigured");
```
Comment (alexcandidos) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/AnalyticsCloudConnectionTest.testcase:178]: hi @ViniLopes20 , I would suggest renaming `list1` to `sectionTitleList`, what do you think?

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/AnalyticsCloudConnectionTest.testcase [hunk 2]
```diff
@@ -193,13 +253,37 @@ definition {
 		}
 
 		task ("Assert the data source status") {
-			ACSettings.assertDataSourceStatus(status = "Active");
+			var i = "0";
```
Comment (alexcandidos) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/AnalyticsCloudConnectionTest.testcase:256]: hi @ViniLopes20 , I would suggest renaming this task to:
"Assert the data source Status: Active, Sites: Configured, Contacts: Configured"

This way will make clear to the reader what is the expected result for this step.

What do you think?

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/AnalyticsCloudConnectionTest.testcase [hunk 3]
```diff
@@ -193,13 +253,37 @@ definition {
...
+				var i = StringUtil.valueOf("${i}");
+			}
 		}
 
 		task ("Disconnect data source and assert the data source status") {
 			ACSettings.disconnectDataSource();
 
-			ACSettings.assertDataSourceStatus(status = "Disconnected");
+			var i = "0";
+
```
Comment (alexcandidos) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/AnalyticsCloudConnectionTest.testcase:275]: hi @ViniLopes20 , I would suggest renaming this task to:
`"Assert the data source Status: Disconnected, Sites: Unconfigured, Contacts: Unconfigured"`

This way will make clear to the reader what is the expected result for this step.

What do you think?

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/AnalyticsCloudConnectionTest.testcase [hunk 4]
```diff
@@ -174,10 +174,70 @@ definition {
...
-			var propertyName = ACDXPSettings.connectDXPtoAnalyticsCloud();
+		var list1 = ListUtil.newListFromString("Current Status,Synced Sites,Synced Contacts");
+		var statusSyncSite = ListUtil.newListFromString("Active,Configured,Unconfigured");
+		var statusSyncContact = ListUtil.newListFromString("Active,Configured,Configured");
+		var statusDisconnect = ListUtil.newListFromString("Disconnected,Unconfigured,Unconfigured");
+		var size = ListUtil.size("${list1}");
 
-			var dataSource = StringUtil.replace("${propertyName}", " Combined Property", "");
+		task ("Connect DXP to AC, create a property and sync site") {
+			ACDXPSettings.goToInstanceSettingsAC();
```
Comment (alexcandidos) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/AnalyticsCloudConnectionTest.testcase:184]: hi @ViniLopes20 , I would suggest renaming this task to:
"Connect DXP to AC, create a property, and sync the site only"

This way will make clear to the reader that you are only syncing the site.

What do you think?

**Review by alexcandidos — APPROVED** [review]

**Comment by alexcandidos** [comment]
ci:forward

**Pull request closed without merging**